website/siteConfig.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+const siteConfig = {
+  title: "Redux", // Title for your website.
+  tagline: "Redux Documentation",
+  url: "https://redux.js.org", // Your website URL
+  baseUrl: "/", // Base URL for your project */
+  // For github.io type URLs, you would set the url and baseUrl like:
+  //   url: 'https://facebook.github.io',
+  //   baseUrl: '/test-site/',
+
+  // Used for publishing and more
+  projectName: "redux",
+  organizationName: "reduxjs",
+  // For top-level user or org sites, the organization is still the same.
+  // e.g., for the https://JoelMarcey.github.io site, it would be set like...
+  //   organizationName: 'JoelMarcey'
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    { doc: "introduction/core-consepts", label: "Docs" },
+    { href: "https://discord.gg/0ZcbPKXt5bZ6au5t", label: "Discord" },
+    { href: "https://stackoverflow.com/questions/tagged/redux", label: "Need help?" },
+  ],
+
+  /* path to images for header/footer */
+  headerIcon: "img/redux.svg",
+  footerIcon: "img/redux.svg",
+  favicon: "img/favicon/favicon.ico",
+
+  /* Colors for website */
+  colors: {
+    primaryColor: "#764ABC",
+    secondaryColor: "#764ABC",
+  },
+
+  /* Custom fonts for website */
+  /*
+  fonts: {
+    myFont: [
+      "Times New Roman",
+      "Serif"
+    ],
+    myOtherFont: [
+      "-apple-system",
+      "system-ui"
+    ]
+  },
+  */
+
+  // This copyright info is used in /core/Footer.js and blog RSS/Atom feeds.
+  copyright: `Copyright © ${new Date().getFullYear()} Your Name or Your Company Name`,
```

Comment:
Need to fill this in.

Reply:
What do you want it to say? "Redux JS"?

Reply:
As per the license (which covers the existing docs), `Copyright (c) 2015-present Dan Abramov`.

Reply:
Actually, maybe we should get a little more specific. Dan doesn't own copyright for things he didn't contribute. `Copyright (c) 2015-present Dan Abramov and the Redux documentation authors.`

Reply:
That's nice. Fixed now.